tests/test/scheduler/test_executor.cpp

```diff
@@ -430,94 +448,6 @@ TEST_CASE_METHOD(TestExecutorFixture,
     }
 }
 
-TEST_CASE_METHOD(TestExecutorFixture,
-                 "Test executing remote chained threads",
-                 "[executor]")
```

Comment:
This test is fiddly and fragile as it's trying to recreate a remote execution. Everything it checks is now covered in the distributed tests which are much cleaner, so I've removed it.